ReactSkia/sdk/CurlNetworking.cpp

```diff
@@ -300,7 +300,8 @@ bool CurlNetworking::sendRequest(shared_ptr<CurlRequest> curlRequest, folly::dyn
     curl_easy_setopt(curl, CURLOPT_PROGRESSDATA, curlRequest.get() );
     curl_easy_setopt(curl, CURLOPT_PROGRESSFUNCTION, progressCallbackCurlWrapper);
   }
-  setHeaders(curlRequest, headers);
+  if(headers != nullptr)
```

Comment:
should we also add the check for "data" variable also , before calling preparePostRequest